roofit/roofitcore/test/TestStatistics/testLikelihoodGradientJob.cpp

```diff
@@ -447,15 +437,14 @@ TEST_F(LikelihoodSimBinnedConstrainedTest, ConstrainedAndOffset)
    // we cannot always expect exactly equal results for fits with likelihood
    // offsetting enabled. See also the LikelihoodSerialSimBinnedConstrainedTest.
    // ConstrainedAndOffset test case in testLikelihoodSerial.
-#define EXPECT_NEAR_REL(a, b, c) EXPECT_NEAR(a, b, std::abs(a *c))
-   EXPECT_NEAR_REL(minNll_nominal, minNll_GradientJob, 1e-4);
+   EXPECT_FLOAT_EQ(minNll_nominal, minNll_GradientJob);
    EXPECT_NEAR(edm_nominal, edm_GradientJob, 1e-5);
-   EXPECT_NEAR_REL(alpha_bkg_A_nominal, alpha_bkg_A_GradientJob, 1e-4);
-   EXPECT_NEAR_REL(alpha_bkg_A_error_nominal, alpha_bkg_A_error_GradientJob, 1e-4);
-   EXPECT_NEAR_REL(alpha_bkg_B_nominal, alpha_bkg_B_GradientJob, 1e-4);
-   EXPECT_NEAR_REL(alpha_bkg_B_error_nominal, alpha_bkg_B_error_GradientJob, 1e-4);
-   EXPECT_NEAR_REL(mu_sig_nominal, mu_sig_GradientJob, 1e-4);
-   EXPECT_NEAR_REL(mu_sig_error_nominal, mu_sig_error_GradientJob, 1e-4);
+   EXPECT_FLOAT_EQ(alpha_bkg_A_nominal, alpha_bkg_A_GradientJob);
```

Comment:
Good suggestion @guitargeek, I will do that.

Originally, btw, I wanted `EXPECT_EQ` for all these things, but this has unfortunately become unfeasible for me. I still think that it should be possible, since all we're doing is testing calculations on the same hardware (as opposed to e.g. comparing CPU vs GPU calculations, where really different floating point units may be present). Perhaps some brave student can pick this up later ;)

Reply:
Oh and @lmoneta: the 1e-4 tolerance was actually necessary because of the bugs fixed here. They were `EXPECT_FLOAT_EQ` before that, I think.